### ISSUE TITLE
There is some error in the code

### ISSUE THREAD

**Issue by grootste** [mnahinkhan/Chess#2]
1352, in <module>
    (square_width*6,square_height*2))
TypeError: integer argument expected, got float

**Comment by ProCoder183** [comment]
transform it into integer by int()

**Comment by mnahinkhan** [comment]
This is likely an issue caused by running with Python 3 instead of 2

**Comment by papagooo** [comment]
Who is running Python 2?

**Comment by mnahinkhan** [comment]
> Who is running Python 2?

Indeed, Python 2 is outdated now. Some of the forks to this project have converted it to Python 3 compatible code, if you are interested :)

**Comment by papagooo** [comment]
just downloaded your V3 updated.txt. First attempt looks like it is working. Impressiv work.

**Comment by mnahinkhan** [comment]
> just downloaded your V3 updated.txt. First attempt looks like it is working. Impressiv work.

Interesting, It's actually not my code, but someone else wrote that in the other Issue thread in which you probably found it. Thank you for your kindness though, hope you enjoy!
